src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -269,10 +275,17 @@ export const ArtistAuctionResultsRefetchContainer = createRefetchContainer(
     const { startAt, endAt } =
       props.artist.auctionResultsConnection?.createdYearRange ?? {}
 
+    const { match } = useRouter()
+    const filters = paramsToCamelCase(match && match.location.query)
```

Comment:
We need to get the filters from the URL query to make searching from the Price Database landing page possible.

Reply:
#superminor - `match?.location.query` might work just as well